sdk/storage/azure-storage-files-datalake/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-storage-blobs-cpp)
+find_dependency(azure-storage-blobs-cpp "12.0.0")
```

Comment:
@Jinming-Hu, should this be `12.1.0` as in your PR?

Reply:
I updated to `12.1.0`, I can revert back to `12.0.0` as it was initially. FWIW, I was able to compile with `12.0.0`.

Reply:
Yes, 12.1.0.